testing/test_checkpoints.py

```diff
@@ -5,50 +5,62 @@
 from neuraxle.pipeline import ResumablePipeline
 from neuraxle.steps.misc import FitTransformCallbackStep, TapeCallbackFunction
 
+FULL_DATA_CHECKPOINTER_WRAPPER = 'FullDataCheckpointerWrapper'
+
 
 def test_resumable_pipeline_with_checkpoint_fit_transform_should_save_data_inputs(tmpdir):
     test_case = create_checkpoint_test_case(tmpdir)
 
     pipeline, outputs = test_case.pipeline.fit_transform([0, 1], [1, 2])
 
-    assert os.path.exists(os.path.join(tmpdir, 'ResumablePipeline', 'checkpoint', 'di', '0.pickle'))
-    assert os.path.exists(os.path.join(tmpdir, 'ResumablePipeline', 'checkpoint', 'di', '1.pickle'))
+    assert os.path.exists(
+        os.path.join(tmpdir, 'ResumablePipeline', 'checkpoint', FULL_DATA_CHECKPOINTER_WRAPPER, 'di', '0.pickle'))
```

Comment:
Does this really work? 
I expected to see different checkpoints for the fit, transform, and fit_transform wrappers. Are you sure you don't miss a subfolder here? Because the context might or should push the subpath of the fit or transform or fit_transform wrappers to the pickles, which would separate them. 

I might be wrong. What will work?

Reply:
this is an old test without wrappers. resolving